ui/page/privacy/setup_privacy_page.go

```diff
@@ -59,72 +51,86 @@ func NewSetupPrivacyPage(l *load.Load, wallet *dcr.Asset) *SetupPrivacyPage {
 									title := pg.Theme.H6(values.String(values.StrStakeShuffle))
-									subtitle := pg.Theme.Body1(values.String(values.StrSetUpPrivacy))
-
-									title.Alignment, subtitle.Alignment = text.Middle, text.Middle
-
+									title.Alignment = text.Start
+									return layout.Inset{
+										Left: values.MarginPadding24,
+									}.Layout(gtx, title.Layout)
+								})
+							}),
+							layout.Rigid(func(gtx C) D {
+								return layout.Inset{
+									Top:    values.MarginPadding24,
+									Bottom: values.MarginPadding24,
+								}.Layout(gtx, func(gtx C) D {
+									return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+										layout.Rigid(func(gtx C) D {
+											return layout.Inset{
+												Left: values.MarginPadding5,
+											}.Layout(gtx, pg.Theme.Icons.TransactionFingerprint.Layout48dp)
+										}),
+										layout.Rigid(pg.Theme.Icons.ArrowForward.Layout24dp),
+										layout.Rigid(func(gtx C) D {
+											return pg.Theme.Icons.Mixer.LayoutSize(gtx, values.MarginPadding120)
+										}),
+										layout.Rigid(pg.Theme.Icons.ArrowForward.Layout24dp),
+										layout.Rigid(func(gtx C) D {
+											return layout.Inset{
+												Left: values.MarginPadding5,
+											}.Layout(gtx, pg.Theme.Icons.TransactionsIcon.Layout48dp)
+										}),
+									)
+								})
+							}),
+							layout.Rigid(func(gtx C) D {
+								return layout.Inset{
+									Left:  values.MarginPadding24,
+									Right: values.MarginPadding24,
+								}.Layout(gtx, func(gtx C) D {
+									introA := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleIntro))
+									introB := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleIntroDesc))
+									introC := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleIntroSubDesc))
+									introA.Alignment, introB.Alignment, introC.Alignment = text.Middle, text.Middle, text.Middle
 									return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
-										layout.Rigid(title.Layout),
+										layout.Rigid(introA.Layout),
 										layout.Rigid(func(gtx C) D {
-											return layout.Inset{Top: values.MarginPadding10}.Layout(gtx, subtitle.Layout)
+											return layout.Inset{Top: values.MarginPadding20}.Layout(gtx, introB.Layout)
+										}),
+										layout.Rigid(func(gtx C) D {
+											return layout.Inset{Top: values.MarginPadding20}.Layout(gtx, introC.Layout)
 										}),
 									)
-								}),
-							)
-						})
+								})
+							}),
+						)
 					})
-				}),
-				layout.Rigid(func(gtx C) D {
-					gtx.Constraints.Min.X = gtx.Constraints.Max.X
-					return layout.UniformInset(values.MarginPadding30).Layout(gtx, pg.toPrivacySetup.Layout)
-				}),
-			)
-		})
+				})
+			}),
+			layout.Rigid(func(gtx C) D {
+				gtx.Constraints.Min.X = gtx.Constraints.Max.X
+				return layout.UniformInset(values.MarginPadding30).Layout(gtx, pg.toPrivacySetup.Layout)
+			}),
+		)
 	})
 }
```

Comment:
Please remove unnecessary code, you can try the code below

```
func (pg *SetupPrivacyPage) privacyIntroLayout(gtx C) D {
	return pg.Theme.Card().Layout(gtx, func(gtx C) D {
		gtx.Constraints.Min.X = gtx.Constraints.Max.X
		return layout.Inset{
			Top:    values.MarginPadding22,
			Bottom: values.MarginPadding22,
			Left:   values.MarginPadding24,
			Right:  values.MarginPadding24,
		}.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
			return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
				layout.Rigid(func(gtx C) D {
					return layout.W.Layout(gtx, func(gtx C) D {
						return pg.Theme.H6(values.String(values.StrStakeShuffle)).Layout(gtx)
					})
				}),
				layout.Rigid(func(gtx C) D {
					return layout.Inset{
						Top:    values.MarginPadding24,
						Bottom: values.MarginPadding24,
					}.Layout(gtx, func(gtx C) D {
						return layout.Center.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
							return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
								layout.Rigid(func(gtx C) D {
									return layout.Inset{
										Left: values.MarginPadding5,
									}.Layout(gtx, pg.Theme.Icons.TransactionFingerprint.Layout48dp)
								}),
								layout.Rigid(pg.Theme.Icons.ArrowForward.Layout24dp),
								layout.Rigid(func(gtx C) D {
									return pg.Theme.Icons.Mixer.LayoutSize(gtx, values.MarginPadding120)
								}),
								layout.Rigid(pg.Theme.Icons.ArrowForward.Layout24dp),
								layout.Rigid(func(gtx C) D {
									return layout.Inset{
										Left: values.MarginPadding5,
									}.Layout(gtx, pg.Theme.Icons.TransactionsIcon.Layout48dp)
								}),
							)
						})
					})
				}),
				layout.Rigid(func(gtx C) D {
					return layout.Inset{
						Left:  values.MarginPadding24,
						Right: values.MarginPadding24,
					}.Layout(gtx, func(gtx C) D {
						introA := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleIntro))
						introB := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleIntroDesc))
						introC := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleIntroSubDesc))
						introA.Alignment, introB.Alignment, introC.Alignment = text.Middle, text.Middle, text.Middle
						return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
							layout.Rigid(introA.Layout),
							layout.Rigid(func(gtx C) D {
								return layout.Inset{Top: values.MarginPadding20}.Layout(gtx, introB.Layout)
							}),
							layout.Rigid(func(gtx C) D {
								return layout.Inset{Top: values.MarginPadding20}.Layout(gtx, introC.Layout)
							}),
						)
					})
				}),
				layout.Rigid(func(gtx C) D {
					return layout.UniformInset(values.MarginPadding30).Layout(gtx, pg.toPrivacySetup.Layout)
				}),
			)

		})
	})
}
```

Reply:
Ok.